features/FEATURE_BLE/targets/TARGET_NORDIC/TARGET_NRF5/source/btle/btle.cpp

```diff
@@ -126,7 +126,7 @@ error_t btle_init(void)
      * Characteristics turned off, then clients are allowed to cache attribute
      * handles making applications simpler on both sides.
      */
-    static const bool IS_SRVC_CHANGED_CHARACT_PRESENT = true;
+    static const bool IS_SRVC_CHANGED_CHARACT_PRESENT = false;
```

Comment:
how is this change relevant to `error_t` ? separate commit or PR?

Reply:
That was not supposed to be changed, thanks for catching that.
My fault for working on multiple issues at once and trying to split them up in git.